data/campaigns/Son_Of_The_Black_Eye/scenarios/09_Shan_Taum_the_Smug.cfg

```diff
@@ -200,7 +200,7 @@
         [/message]
         [message]
             speaker="Pirk"
-            message= _ "Go defend Prestim, Kapou’e. In the meantime, now that the Council is complete again, we will decide. We may have to form the Great Horde again, and give you the leadership of it."
+            message= _ "Go defend Prestim, Kapou’e. In the meantime, now that the Council is complete again, we will decide. We may have to form the Great Horde again and make you its leader."
```

Comment:
I think, when someone is speaking orally, they are likely not to be concise enough to say "re-form X" instead of "form X again". Like how people often say "I'm not going back there again" instead of "im not returning there".

Reply:
No worries, just thought it sounded better to me in the moment. I'm not too fussy either way.